src/guake/gconfhandler.py

```diff
@@ -364,7 +364,6 @@ def reload_globals(self, client, connection_id, entry, data):
         method. If more global hotkeys should be added, just connect
         the gconf key to the watch system and add.
         """
-        self.guake.hotkeys.unbind_all()
```

Comment:
it is just a matter of manually call unbind(key) for each registered key. We can make a wrapper in guake that handles this. If you want to do, feel free!